src/notifications/generate.ts

```diff
@@ -307,4 +310,14 @@ export const generateNotificationMap: Record<
       )
       .targetUrl(generateDevCard)
       .uniqueKey(ctx.userIds[0]),
+  source_post_added: (
+    builder,
+    ctx: NotificationPostContext & NotificationDoneByContext,
+  ) =>
+    builder
+      .referenceSource(ctx.source)
+      .icon(NotificationIcon.Bell)
+      .avatarSource(ctx.source)
+      .targetSource(ctx.source)
+      .uniqueKey(ctx.post.id),
```

Comment:
Unique key so that user gets notified whenever new post is added. If not this user would be notified only once.

Reply:
Not needed anymore because now we reference post for attachment.